README.md

```diff
@@ -24,6 +24,8 @@ pip install -U polygon-api-client
 
 To get started, please see the [Getting Started](https://polygon-api-client.readthedocs.io/en/latest/Getting-Started.html) section in our docs, view the [examples](./examples) directory for code snippets, or view the [blog post with videos](https://polygon.io/blog/polygon-io-with-python-for-stock-market-data/) to learn more.
 
+When using a free API key, please be mindful that it supports up to 5 API calls per minute. Exceeding this limit will throw an error message related to rate limits. For uninterrupted access and higher limits, consider upgrading to our [Starter plan](https://polygon.io/pricing), which offers unlimited API calls. This ensures a smoother experience, especially when working with real-time data or building applications that require higher request volumes.
```

Comment:
maybe a nit idk, should we be a bit more vague with wording in here and let the link do the talking? I hesitate to put explicit claims like "5 req/s" and "starter plan has unlimited api calls" in here. It's true now but if we change things in the future there's little chance we'll remember to update this README. 

I like the idea of adding a note but maybe we can say something like "free API keys are restricted in how many req/s and the type of data they can access, see our pricing page for details on API key restrictions by product tier" (but in a more eloquent way)

Reply:
Fixed. Thanks and this totally makes sense re: not needing to keep these docs in sync if something changes. Just send them to the docs for more info.